packages/realm/src/app-services/MongoDBCollection.ts

```diff
@@ -608,10 +616,15 @@ export class MongoDBCollection<T extends Document> {
 
   /**
    * Updates multiple documents matching the provided filter in this collection.
-   * @param filter A filter applied to narrow down the result.
-   * @param update The new values for the documents.
-   * @param options Additional options to apply.
-   * @returns The result.
+   * @param filter - A filter applied to narrow down the result.
+   * @param update - The new values for the documents.
+   * @param options - Additional options to apply.
+   * @returns A promise that resolves to an object containing:
+   * {
+   *  matchedCount: number;
+   *  modifiedCount: number;
+   *  upsertedId: IdType | undefined;
+   * }
```

Comment:
```suggestion
   * ```
   * {
   *  matchedCount: number;
   *  modifiedCount: number;
   *  upsertedId: IdType | undefined;
   * }
   * ```
```